newsletter-bundle/contao/dca/tl_module.php

```diff
@@ -58,23 +58,17 @@
 
 $GLOBALS['TL_DCA']['tl_module']['fields']['nl_subscribe'] = array
 (
+	'default'                 => static fn(): string => $GLOBALS['TL_LANG']['tl_module']['text_subscribe'][1],
```

Comment:
this doesn't really need to be a callback, but should probably null-check the label